app/src/main/java/swati4star/createpdf/util/FileSortUtils.java

```diff
@@ -7,12 +7,12 @@
 public class FileSortUtils {
 
     // Sorting order constants
-    public static final int NAME_INDEX = 0;
-    public static final int DATE_INDEX = 1;
-    public static final int SIZE_INCREASING_ORDER_INDEX = 2;
-    public static final int SIZE_DECREASING_ORDER_INDEX = 3;
+    public final int NAME_INDEX = 0;
```

Comment:
shouldnt these be static?

Reply:
As they are class members and we are having a static instance. I thought accessing these variables through static instance is good.